src/python/lib/setup.py

```diff
@@ -55,7 +55,8 @@
         'six',
     ],
     package_data={
-        'clusterfuzz': ['lib-config/**',],
+        'clusterfuzz': ['lib-config/*', 'lib-config/**/*'],
```

Comment:
is 'lib-config/*' needed still ?

Reply:
Yes unfortunately both are needed. lib-config/**/* only includes files that are at least 1 directory deep.